cves/CVE-2018-6621.yml

```diff
@@ -326,8 +345,9 @@ lessons:
     applies:
     note:
   complex_inputs:
-    applies:
-    note:
+    applies: true
+    note: |
+      Video frames are complex input, especially with a lossless codec like UT.
```

Comment:
The term 'lossless codec' I would say is not common knowledge to the target audience. Maybe a quick explanation or a different term to clarify things.